src/main/java/org/littil/api/ApplicationLifeCycle.java

```diff
@@ -149,4 +157,44 @@ private static Location createLocationForDevUser(String address, String postalCo
         return location;
     }
 
+    private Map<String, Location> getDevUsersFromFile() {
+        List<String> devUserData = readDevUsersFromFile();
+        return devUserData.stream() //
+                .filter(s -> !s.contains("#"))
+                .map(s -> s.split(","))
+                .filter(a -> a.length ==  3)
+                .filter(a -> isValidString(a[0],EMAIL_REGEX))
+                .filter(a -> isValidString(a[2],ZIPCODE_REGEX))
+                .collect(Collectors.toMap(
+                        a -> a[0],
+                        a -> createLocationForDevUser(a[1], a[2])
+                ));
+    }
+
+    private List<String> readDevUsersFromFile() {
+        List<String> devUserData = new ArrayList<>();
+        try (BufferedReader br = new BufferedReader( new FileReader(devUserDataFile))){
```

Comment:
Why not just use `Files.readAllLines()`? The data will be stored in a `List<>` in memory, so memory size should not be an issue.

Reply:
changed to Files.readAllLines()